docs(optimize): implement gramatical suggestions (#2201)
camunda/camunda-platform-docs: optimize/components/userguide/creating-reports.md

5. Set up and customize your report. Begin by naming your report in the text box at the top of the page.
6. In the gray text box to the right, confirm your data source, and select what you’d like to review from the process. You can also group by topics such as duration or start date.
7. If you’d like, filter the process instance or flow nodes. For example, you can filter by duration, only viewing process instances running for more than seven days.
8. Additionally, a **Report** can be given a description which is displayed below the **Report** name. Using the **Add/Edit** button, you can add, edit, or remove the description. You are limited to a 400-character plain text description.

![Add description modal](./img/report-descriptionModal.png)

> 1. You have the option to view particular sets of data from the instance (like instance count or absolute value) by selecting the gear icon to the left of your data customization. You can also choose to visualize your data in the box beneath **Visualization** (i.e. bar chart, pie chart, etc.). Once you’ve made your selections, click **Save**.

Reviewer: The ordering looks off here, but running the docs locally displays a 9, as expected. This is fine. (I only caught it as I was going to approve the PR!)

5. Set up and customize your report. Begin by naming your report in the text box at the top of the page.
6. In the gray text box to the right, confirm your data source, and select what you’d like to review from the process. You can also group by topics such as duration or start date.
7. If you’d like, filter the process instance or flow nodes. For example, you can filter by duration, only viewing process instances running for more than seven days.

> 8. Additionally, a **Report** can be given a description which is displayed below the **Report** name. Using the **Add/Edit** button, you can add, edit, or remove the description. The text editor allows you to add plain text description with a limit of 400 characters.

Reviewer: ```suggestion
8. Additionally, a **Report** can be given a description which is displayed below the **Report** name. Using the **Add/Edit** button, you can add, edit, or remove the description. You are limited to a 400-character plain text description.
```

This is a bit nit picky, but felt more concise.

Reviewer: Actually, I will make the review changes in a separate commit and make this easier for everyone 🙂
